packages/compass/package.json

```diff
@@ -231,6 +231,7 @@
     "@mongodb-js/my-queries-storage": "^0.22.0",
     "@mongodb-js/prettier-config-compass": "^1.2.0",
     "@mongodb-js/sbom-tools": "^0.7.0",
+    "@mongodb-js/signing-utils": "^0.3.7",
```

Comment:
So the bug was that signtool.go is doing `node -e 'require("@mongodb-js/signing-utils")'` from the packages/compass folder and that node module doesn't exist there.

In the past it probably worked because the dep was hoisted and then it broke because now it is not.

Reply:
I think doing something like `npx run sign -w @mongodb-js/hadron-build -- path-to-file` from signtool.go would be a cleaner fix. I'll try that in a follow-up.